spec.html

```diff
@@ -17188,149 +17058,89 @@ <h1>Static Semantics: SV ( ): a String</h1>
             The SV of <emu-grammar>EscapeSequence :: `0`</emu-grammar> is the String value consisting of the code unit 0x0000 (NULL).
           </li>
           <li>
-            The SV of <emu-grammar>CharacterEscapeSequence :: SingleEscapeCharacter</emu-grammar> is the String value consisting of the code unit whose numeric value is determined by the |SingleEscapeCharacter| according to <emu-xref href="#table-string-single-character-escape-sequences"></emu-xref>.
+            The SV of <emu-grammar>CharacterEscapeSequence :: SingleEscapeCharacter</emu-grammar> is the String value consisting of the single code unit associated with |SingleEscapeCharacter| according to <emu-xref href="#table-string-single-character-escape-sequences"></emu-xref>.
           </li>
         </ul>
         <emu-table id="table-string-single-character-escape-sequences" caption="String Single Character Escape Sequences" oldids="table-34">
           <table>
             <tr>
               <th>
-                Escape Sequence
-              </th>
-              <th>
-                Code Unit Value
+                |SingleEscapeCharacter|
```

Comment:
Note that a `|SingleEscapeCharacter|` is a single character and *does not include the preceding backslash*. So it's a mismatch to have `|SingleEscapeCharacter|` as the column head and then (e.g.) `\b` below it. (The status quo uses "Escape Sequence" as the column head, which is not a defined term. You'd have to go up to `|DoubleStringCharacter|` and `|SingleStringCharacter|` to get a nonterminal that actually includes the backslash.)

The simplest fix would be to delete the backslashes from the data cells (as in `Table 61: ControlEscape Code Point Values`), although that loses the visual cue that they're 'escape sequences'.

Alternatively, you could insert a backslash into the header cell, but that's a bit dodgy, since:
- ``` `\` |SingleEscapeCharacter| ``` doesn't occur in the grammar, and
- the prose `associated with |SingleEscapeCharacter|` wouldn't be quite right.

Reply:
I'll just remove the backslash.

Reply:
Actually, I think I'll replace it with a code point descriptor.

Reply:
Well, that's valid, but I'm not sure it's an improvement (over just removing the backslashes). The definition of _SingleEscapeCharacter_ is **one of ' " \ b f n r t v**, so it seems like the natural approach would be to use those characters rather than code point descriptors.

Reply:
I'm fine with changing to just the single character if the other editors prefer.